scripts/calculateRewards/celoPG.ts

```diff
@@ -102,22 +102,10 @@ function parseArgs() {
       type: 'number',
       default: 1,
     })
-    .option('excludelist', {
-      description:
-        'Comma-separated list of CSV files with excluded addresses (e.g., file1.csv,file2.csv)',
-      type: 'array',
-      default: [],
-      coerce: (arg: string[]) => {
-        return arg
-          .flatMap((s) => s.split(',').map((item) => item.trim()))
-          .filter(Boolean)
-      },
-    })
-    .option('fail-on-exclude', {
-      description:
-        'Fail if any of the excluded addresses are found in the referral events',
-      type: 'boolean',
-      default: false,
+    .option('builder-allowlist-file', {
```

Comment:
i figure that if there is an inclusion list from karma gap, then we don't really need our own exclusion list on top of that.